### ISSUE TITLE
Extreme High Server CPU Load after 1.82 Upgrade

### ISSUE THREAD

**Issue by WebSavvyGuy** [engintron/engintron#462]
Hi

We have a few servers that went through the Engintron Upgrade to 1.82 yesterday.  

1 server is a regular server hosting some websites and that is running smooth after the upgrade.

However, the other 2 servers have a java/flash application  and ever since the upgrade it has been building up the CPU load times by a significant amount.   I am talking over 200 CPU load times that I can see in WHM.

It is causing alot of strain on the server.  I tried killing the PID's causing this and even restarting the server but it just starts to build the CPU load back up again as users continue to use the application.

This only started after upgrading to 1.82   

Is there a way i can revert back to 1.80 (which is what i was on, i didnt upgrade to 1.81) which had no issues with this software application.   Unless its a very weird coincidence, I believe the upgrade has caused this issue.

Can you think of any changes between 1.80 and 1.82 that might cause this?   @fevangelou

**Comment by WebSavvyGuy** [comment]
Just to follow up, I just disabled Engintron but the CPU load continues to go up so it might not be related afterall.

Is it possible that anything was installed on the server due to the upgrade?  Something that would not be undone from clicking "disable Engintron"?

If not, then its another issue.  The only weird thing is it all started almost immediately after the upgrade.

**Comment by fevangelou** [comment]
Of course not.

**Comment by fevangelou** [comment]
Do a reboot. Works magic sometimes if processes get stuck.

**Comment by WebSavvyGuy** [comment]
Yeah, done already with both Engintron "disabled" and active.  No luck there.  So baffling what happened.   Wish you offered services to actually take a look at the server.

**Comment by fevangelou** [comment]
I do offer such services.

https://github.com/engintron/engintron#more-info

**Comment by fevangelou** [comment]
Apparently @WebSavvyGuy does have increased CPU load but it's not caused by Engintron.

Closing for now.

**Comment by WebSavvyGuy** [comment]
To be honest i have not ruled it out.   I did disable Engintron and it still had a high CPU load but I have not gotten to the bottom of all this yet.   Such questions as to why this all started after updating Engintron to 1.82 from 1.80 still exists.  Happened on TWO servers also.     It is entirely possible an update to 1.82 has changed some system files.

@fevangelou    My question is if i revert back to 1.80 would it fix anything that may have been changed?
